subsys/zigbee/Kconfig

```diff
@@ -11,6 +11,7 @@ menuconfig ZIGBEE
 	select PM_SINGLE_IMAGE
 	select NET_L2_ZIGBEE
 	select NETWORKING
+	select NET_PKT_TXTIME
```

Comment:
we can select NET_PKT_TXTIME permanently in Kconfig or only when needed in prj.conf